src/main/scala/TwitterStream.scala

```diff
@@ -0,0 +1,31 @@
+import org.apache.spark.streaming.twitter._
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import TwitterSetup.setupTwitter
+import twitter4j.FilterQuery
+import com.typesafe.scalalogging.LazyLogging
+
+object TwitterStream extends LazyLogging {
+  def main(args: Array[String]) {
+
+    // Configure Twitter credentials using twitter.txt
+    setupTwitter()
+    val ssc = new StreamingContext("local[*]", "PrintTweets", Seconds(1))
+
+    // Create a DStream from Twitter using our streaming context
+    val filteredQuery = new FilterQuery().track("blockchain", "btc", "crypto", "cryptocurrency",
+      "bitcoin", "ICO", "Ethereum", "altcoin", "dogecoin", "cryptomemes")
+
+    //val tweets = TwitterUtils.createStream(ssc, None)
+    val tweets = TwitterUtils.createFilteredStream(ssc, None, Some(filteredQuery))
+
+    // Now extract the text of each status update into RDD's using map()
+    val statuses = tweets
```

Comment:
all of the comments are actually either redundant or can be replaced by more self-descriptive code/API namings.
please read `chapter 4 - Comments` of this book:  [Clean Code](http://ricardogeek.com/docs/r_clean_code.pdf)

NOTE: the refactoring has smaller priority than the rest of action items, but it's vital to learn software development best practices in order to be able to write production quality code (bugless, maintainable and performant).

So, going forward,  please strive to writing self-documenting code with commenting only when it makes sense.